Public/Standalone/Set-PDFForm.ps1

```diff
@@ -1,32 +1,62 @@
 function Set-PDFForm {
-    [CmdletBinding()]
+   [CmdletBinding()]
     param(
-        [string] $SourceFilePath,
-        [string] $DestinationFilePath,
-        [System.Collections.IDictionary]$FieldNameAndValueHashTable
+        [Parameter(Mandatory=$true)]$SourceFilePath,
+        [Parameter(Mandatory=$true)]$DestinationFilePath,
+        [System.Collections.IDictionary]$FieldNameAndValueHashTable,
+        [Switch] $Flatten
     )
 
-    $Script:Reader = [iText.Kernel.Pdf.PdfReader]::new($SourceFilePath)
-    $Script:Writer = [iText.Kernel.Pdf.PdfWriter]::new($DestinationFilePath)
-    $PDF = [iText.Kernel.Pdf.PdfDocument]::new($Script:Reader, $Script:Writer)
+    if ((Test-Path -LiteralPath $SourceFilePath) -and (Test-Path -LiteralPath $SourceFilePath)) {
```

Comment:
You're checking for the same path twice. Don't think it's necessary. Also not necessary to check for DesitnationPath as it most likely won't exists. Also take a look here: https://github.com/EvotecIT/PSWritePDF/blob/cf14d56e1d1c8916643c5f57341dba9099a57253/Public/Standalone/Merge-PDF.ps1#L19 for Convert-Path usage as it will make sure that if Relative path is used it will be converted to proper path. Otherwise the function won't support relative paths.